apps/backend/src/main/resources/api.yaml

```diff
@@ -13,4 +13,108 @@ paths:
     get:
       responses:
         400:
-          description: Bad request
\ No newline at end of file
+          description: Bad request
+components:
+  schemas:
+    ModelDTO:
+      required:
+        - description
+        - name
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+    LocationDTO:
+      required:
+        - description
+        - name
+        - parentLocationId
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+        parentLocationId:
+          type: integer
+          nullable: true
+    ItemDTO:
+      required:
+        - modelId
+        - movementsId
+      properties:
+        id:
+          type: integer
+        modelId:
+          type: integer
+        movementsId:
```

Comment:
`movementsIds` does not seem necessary (right now):
`itemId` is already in `MovementDTO`.

All `MovementDTO` for an item can still be read with something like `movement?where[itemId][$in][]=[1]&where[itemId][$in][]=[2]`

Reply:
Works for me